README.md

```diff
@@ -1 +1,41 @@
-# hw01
\ No newline at end of file
+# hw01
+
+
+
+**Who am I?**
+========
+
+<img src="https://github.com/recalderamirez/hw01/blob/master/LUIS.jpg?raw=true" width="400">
```

Comment:
So I did that, following the Markdown cheatsheet posted in class website (please refer to my pull-request comment above). The problem that I found is that using `![](/path/LUIS.jpg)` results in a really large image. I googled the solution and found that I could write `![](/path/LUIS.jpg){width=30%}` for a 30% of the size. This worked in R when I clicked "preview", but when pushed to GitHub, it didn't show. Therefore I had to use the HTML code. If you have any suggestions to change image size so that it works for both R and Github, let me know. I'd really appreciate it.